test/index.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+/* global describe, it */
+
+const assert = require('assert');
+const fs = require('fs');
+
+const htmlparser = require('htmlparser2');
+const R = require('ramda');
+
+const H = require('..');
+
+const eq = function(actual, expected) {
+  assert.strictEqual(arguments.length, 2);
+  assert.strictEqual(R.toString(actual), R.toString(expected));
+};
+
+// const eq = chai.assert.deepEqual;
+// chai.config.truncateThreshold = 100;
+
+const GOOD_HTML = fs.readFileSync(__dirname + '/good.html', 'utf8');
+
+
+describe('attr', () => {
+
+});
+
+describe('attr', () => {});
+
+describe('children', () => {});
+
+describe('find', () => {});
+
+describe('html', () => {});
+
+describe('is', () => {});
+
+describe('next', () => {});
+
+describe('Node', () => {
+  it('returns Node given htmlparser2 _node', () => {
+    const _node = {
+      name: '!doctype',
+      data: '!DOCTYPE html',
+      type: 'directive',
+      next: null,
+      prev: null,
+      parent: null
+    };
+    const node = H.Node(_node);
+    eq(node.value, _node);
+    eq(node['_@@type'], 'sanctuary-html/Node');
+    eq(R.toString(node), 'Node("<!DOCTYPE html>")');
+    eq(R.equals(node, H.Node(_node)), true);
+  });
+});
+
+describe('parent', () => {});
+
+describe('parse', () => {
+  it('parse well formed html', () => {
```

Comment:
> Do you think stricter assertions are warranted?

How wordy would it be to include the entire value in the form produced by htmlparser2?